dali/python/nvidia/dali/external_source.py

```diff
@@ -255,20 +256,43 @@ def _accepted_arg_count(callable):
     return callable.__code__.co_argcount - implicit_args
 
 def _get_callback_from_source(source, cycle):
```

Comment:
This function should be in the same module as the utilities it uses (_SourceDescirption, _SourceKind, etc) - either move them here or move this function (along with other utilities it depends on) elsewhere.

Reply:
I moved more to the external_source_impl.py.